File: D3_DispatchNet/code/src/user_management/main.cpp
#include "user_management.h"
#include <iostream>
using namespace std;

int main() {
    NetworkManager networkManager("admin", "admin@example.com", "passwordsicura");
    cout << "Username: " << networkManager.getUsername() << endl;
    cout << "Email: " << networkManager.getEmail() << endl;
    cout << "User Type: " << networkManager.getUserType() << endl;

    return 0;
}

File: D3_DispatchNet/code/src/user_management/Main.java
package user_management;

public class Main {
    public static void main(String[] args) {
        NetworkManager networkManager = new NetworkManager("admin", "admin@example.com", "passwordsicura");

        System.out.println("Username: " + networkManager.getUsername());
        System.out.println("Email: " + networkManager.getEmail());
        System.out.println("User Type: " + networkManager.getUserType());
    }
}
